app/node/project/footage/footage.cpp

```diff
@@ -36,6 +36,7 @@
 namespace olive {
 
 const QString Footage::kFilenameInput = QStringLiteral("file_in");
+const QString Footage::kVideoParamsInput = QStringLiteral("video_params");
```

Comment:
Hmm? Footage is (now) a derivative of ViewerOutput which defines `kVideoParamsInput` here: https://github.com/olive-editor/olive/blob/master/app/node/output/viewer/viewer.cpp#L30 (it would fail to compile if it wasn't defined)

Seems more likely the crash is elsewhere and this "dodges" it by making the connection somewhere else.

Reply:
So it is, I missed that.

As far as I can tell if I load an OTIO file with missing media Olive doesn't created a VideoParams widget. When I then relink the footage it tries to refernece something that doesn't exist. I'm not reall sure how to fix this.